articles/azure/azs-identify-global-admins.md

```diff
@@ -0,0 +1,57 @@
+---
+title: How to identify your Global Admins| UKCloud Ltd
```

Comment:
global administrators (unless there's a specific reason for it to be Global Admins)

Reply:
done